@here/olp-sdk-dataservice-api/lib/interactive-api-v1.ts

```diff
@@ -0,0 +1,561 @@
+    /**
+     * Version of API
+     */
+    apiVersion?: string;
+}
+
+export interface Feature extends GeoJSON {
+    /**
+     * The unique identifier of the feature.
+     */
+    id?: string;
+    geometry?: GeoJSON;
+    properties?: { [key: string]: any };
+}
+
+export interface FeatureCollection extends GeoJSON {
+    /**
+     * Features included in the collection.
+     */
+    features: Array<Feature>;
+}
+
+export interface FeatureCollectionIterable extends FeatureCollection {
+    /**
+     * The handle of the next batch.
+     */
+    handle?: string;
+    /**
+     * The token of the next batch.
+     */
+    nextPageToken?: string;
+}
+
+export interface FeatureCollectionModification extends FeatureCollection {
+    /**
+     * List of inserted feature IDs.
+     */
+    inserted?: Array<string>;
+    /**
+     * List of updated features IDs.
+     */
+    updated?: Array<string>;
+    /**
+     * List of deleted features IDs.
+     */
+    deleted?: Array<string>;
+}
+
+/**
+ * The base type for all possible GeoJSON objects.
+ */
+export interface GeoJSON {
+    type: string;
+    /**
+     * Describes the coordinate range of the GeoJSON object.
+     */
+    bbox?: Array<number>;
+}
+
+export interface LineString extends GeoJSON {
+    coordinates?: Array<Array<number>>;
+}
+
+export interface ModelError {
+    title?: string;
+    status?: number;
+    code?: string;
+    cause?: string;
+    action?: string;
+    correlationId?: string;
+}
+
+export interface MultiLineString extends GeoJSON {
+    coordinates?: Array<Array<Array<number>>>;
+}
+
+export interface MultiPoint extends GeoJSON {
+    coordinates?: Array<Array<number>>;
+}
+
+export interface MultiPolygon extends GeoJSON {
+    coordinates?: Array<Array<Array<Array<number>>>>;
+}
+
+export interface Point extends GeoJSON {
+    coordinates?: Array<number>;
+}
+
+export interface Polygon extends GeoJSON {
+    coordinates?: Array<Array<Array<number>>>;
+}
+
+export interface Statistics {
+    count?: StatisticsCount;
+    byteSize?: StatisticsCount;
+    bbox?: StatisticsBbox;
+    geometryTypes?: StatisticsGeometryTypes;
+    properties?: StatisticsProperties;
+    tags?: StatisticsTags;
+    searchable?: string;
+}
+
+export interface StatisticsBbox {
+    value?: Array<number>;
+    estimated?: boolean;
+}
+
+export interface StatisticsCount {
+    value?: number;
+    estimated?: boolean;
+}
+
+export interface StatisticsGeometryTypes {
+    value?: Array<string>;
+    estimated?: boolean;
+}
+
+export interface StatisticsProperties {
+    value?: Array<StatisticsPropertiesValue>;
+    estimated?: boolean;
+}
+
+export interface StatisticsPropertiesValue {
+    key?: string;
+    count?: number;
+    searchable?: boolean;
+}
+
+export interface StatisticsTags {
+    value?: Array<StatisticsTagsValue>;
+    estimated?: boolean;
+}
+
+export interface StatisticsTagsValue {
+    key?: string;
+    count?: number;
+}
+
+type StrBoolean = "true" | "false";
```

Comment:
Some are boolean others not. Above StatisticsPropertiesValue.searchable is boolean. Can we keep consistent and always use `bool` for external, while internally we convert to whatever the REST API needs.